infrastructure/code/code_html.go

```diff
@@ -246,6 +246,26 @@ func getRepoName(commitURL string) string {
 	return tabTitle
 }
 
+func formatExpirationDate(expiration string) string {
+	if expiration == "" {
+		return "No expiration"
+	}
+	parsedDate, err := time.Parse(time.RFC3339, expiration)
+	if err != nil {
+		return expiration // Original string if parsing fails
+	}
+
+	// Calculate the difference in days
+	daysRemaining := int(parsedDate.Sub(time.Now()).Hours() / 24)
+
+	if daysRemaining < 0 {
+		return "Expired"
```

Comment:
I would expect that if the ignore has expired then it doesn't get returned but it doesn't hurt to have this

Reply:
That's true... 🤦‍♀️ 
Thank you for pointing this out 🙏